src/components/Header/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { CustomAvatar } from 'components/shared';
+import React from 'react';
+import avatar1 from 'assets/img/avatar1.svg';
+import avatar2 from 'assets/img/avatar2.svg';
+import avatar3 from 'assets/img/avatar3.svg';
+import avatar4 from 'assets/img/avatar4.svg';
+import header1 from 'assets/img/header-img.svg';
+import styles from './style.module.scss';
+
+export const HeaderComponent = () => (
+	<div>
+		<h1 className={styles.app__title}>
+			<span>Five</span> Leaderboard
+		</h1>
+		<div className={styles.app__header}>
+			<div>
+				<h2 className={styles.app__header_title}>All-time highest scorers</h2>
+				<h3 className={styles.app__header_subtitle}>You can be among the leaders already today</h3>
+				<div className={styles.app__bests}>
+					<div className={styles.app__best}>
+						<CustomAvatar src={avatar1} style={{ height: '59px', width: '60px' }} title="Petr" />
```

Comment:
don't use style props. create a mui style or if the styles of avatars are the same for all in Figma, change the styles in the theme